plugin-template

```diff
@@ -402,6 +403,15 @@ def write_template_section(config, name, plugin_root_dir, verbose=False):
             files_copied += 1
             if verbose:
                 print(f"Copied file: {relative_path}")
+        if config["black"] and destination_relative_path.endswith((".py", ".py.j2")):
+            if destination_relative_path.endswith(".py.j2"):
+                destination = destination_relative_path[: -len(".j2")]
+            else:
+                destination = destination_relative_path
+            destination = os.path.join(plugin_root_dir, destination)
+
+            if os.path.exists(destination):
+                subprocess.run(["black", "--quiet", destination])
```

Comment:
I like the idea. Is it evaluating settings in pyproject.toml here?

Reply:
As pwd is the template checkout, not the plugin checkout, probably not. Should I try to make it to?

Reply:
How? :)
But in the case of bootstrap, it will be fixed in a single "black" commit at some point and everything is good (again).

Updated the code to use `cwd=plugin_root`, so it *should* pickup the settings (untested)

Reply:
> Or just run black on everything after all the templates.

On *everything* or *everything `generate_relative_path_set` returned*? The former would possibly reformat things that were not part of the templating set.

Reply:
Hah, but posting this, I realized, black might not (yet) be installed when `plugin-template` runs